### ISSUE TITLE
rename the 'cis' option to 'usg'

### ISSUE THREAD

**Issue by nmav** [canonical/ubuntu-advantage-client#1820]
As we increase the security compliance content that is provided in the the 'cis' repository, the naming used on the ua client must be updated as well. I propose to update it to:
```
SERVICE       ENTITLED  STATUS    DESCRIPTION
usg           --        —         Security compliance and audit tools
```

while leaving the original name (cis) as an alias for this option. This will allow the existing instructions to work, and instructions for new content will use the new naming. This is something that should be available ~November in 20.04 and the default in 22.04.

(see also https://warthogs.atlassian.net/browse/SP-86 )

**Comment by blackboxsw** [comment]
ongoing work https://warthogs.atlassian.net/browse/SC-576

**Comment by renanrodrigo** [comment]
released in 27.5
